pyproject.toml

```diff
@@ -1,15 +1,15 @@
 [build-system]
 build-backend = "mesonpy"
 requires = [
-    "meson-python>=0.15.0",
+    "meson-python>=0.18.0",
     "Cython>=3.0.6",  # keep in sync with version check in meson.build
 ]
 
 [project]
 name = "numpy"
 version = "2.4.0.dev0"
-# TODO: add `license-files` once PEP 639 is accepted (see meson-python#88)
-license = {file = "LICENSE.txt"}
+license = "BSD-3-Clause"
```

Comment:
Out of curiosity/healthy skepticism, I naturally tried:

```diff
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -8,7 +8,7 @@ requires = [
 [project]
 name = "numpy"
 version = "2.4.0.dev0"
-license = "BSD-3-Clause"
+license = "BSD-20-Clause"
```

and I do indeed see an error when I do an isolated local build:

`meson-python: error: Unknown license: 'bsd-20-clause'`